test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs

```diff
@@ -59,31 +47,57 @@ public void SetupScoringSpeedTests()
         }
 
         [Benchmark]
-        public void TrainTest_Ranking_MSLRWeb10K_RawNumericFeatures_FastTreeRanking()
+        public void TrainTest_Ranking_MSLRWeb10K_RawNumericFeatures_LightGBMRanking()
         {
             string cmd = @"TrainTest test=" + _mslrWeb10k_Validate +
                 " eval=RankingEvaluator{t=10}" +
                 " data=" + _mslrWeb10k_Train +
                 " loader=TextLoader{col=Label:R4:0 col=GroupId:TX:1 col=Features:R4:2-138}" +
-                " xf=HashTransform{col=GroupId} xf=NAHandleTransform{col=Features}" +
-                " tr=FastTreeRanking{}";
+                " xf=HashTransform{col=GroupId}" +
+                " xf=NAHandleTransform{col=Features}" +
+                " tr=LightGBMRanking{}";
 
             using (var environment = new ConsoleEnvironment(verbose: false, sensitivity: MessageSensitivity.None, outWriter: EmptyWriter.Instance))
             {
                 Maml.MainCore(environment, cmd, alwaysPrintStacktrace: false);
             }
         }
+    }
 
-        [Benchmark]
-        public void TrainTest_Ranking_MSLRWeb10K_RawNumericFeatures_LightGBMRanking()
+    [Config(typeof(PredictConfig))]
+    public class RankingTest
```

Comment:
there are `XYZTrain` and `XYZTest` types for Training and Predict benchmarks. Which is a better name from the ML perspective: Test or Predict?

Reply:
`Test` and `Predict` are 2 different ML operations.
The `Test` action is a super set of `Predict`. To do `Test`, you need some test data with the real label/answer in it. Then you go through all the test values and `Predict` on them all. Now you have 2 values for each instance: the `PredictedLabel` and the real actual `Label`, which you can compare, evaluate and get a measure of how well your model performs.

So `Predict` is just one step in `Test`.

Reply:
@eerhardt thanks for the explanation!